.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,59 @@
+name: Integration Test
+on: [push]
+
+jobs:
+  es-node-integration-test:
+    runs-on: self-hosted
+    timeout-minutes: 1440
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.21'
+
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20.x'
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly
+
+      - name: Deploy Contracts
+        run: |
+          cd ../storage-contracts-v1 
+          git checkout integration-test
+          git pull
+          npm install 
+          git submodule init && git submodule update
+          npx hardhat run scripts/deployL2-it.js --network qkc_testnet >> deploy.log
+          echo ES_NODE_CONTRACT_ADDRESS=`cat .caddr` >> "$GITHUB_ENV"
+          echo $ES_NODE_CONTRACT_ADDRESS
+
+      - name: Build and Run Bootnode Node
+        run: |
+          cd ../es-node
+          make
+          ./run-l2-it-rpc.sh > es-node-it-bootnode.log&

```

Comment:
Why the bootnode is started as a RPC node?

Reply:
the other node covers mining, syncing, and encoding/decoding parts, this bootnode can be run as RPC to cover more.